target-platforms/packages/portal-7.4-ga1/config.json

```diff
@@ -0,0 +1,197 @@
+				"metal-router": "*",
+				"metal-scrollspy": "*",
+				"metal-soy": "*",
+				"metal-soy-bundle": "*",
+				"metal-state": "*",
+				"metal-storage": "*",
+				"metal-structs": "*",
+				"metal-throttle": "*",
+				"metal-toggler": "*",
+				"metal-uri": "*",
+				"metal-useragent": "*",
+				"metal-web-component": "*",
+				"querystring": "*",
+				"xss-filters": "*"
+			},
+			"frontend-js-spa-web": {
+				"senna": "*"
+			},
+			"frontend-js-web": {
+				"/": "*",
+				"@liferay/amd-loader": "*",
+				"lodash.escape": "*",
+				"lodash.groupby": "*",
+				"lodash.isequal": "*",
+				"lodash.memoize": "*",
+				"lodash.unescape": "*",
+				"svg4everybody": "*",
+				"uuid": "*"
+			},
+			"frontend-taglib-chart": {
+				"clay-charts": "*"
+			},
+			"frontend-taglib-clay": {
+				"@clayui/alert": "*",
+				"@clayui/autocomplete": "*",
+				"@clayui/badge": "*",
+				"@clayui/breadcrumb": "*",
+				"@clayui/button": "*",
+				"@clayui/card": "*",
+				"@clayui/charts": "*",
+				"@clayui/color-picker": "*",
+				"@clayui/css": "*",
+				"@clayui/data-provider": "*",
+				"@clayui/date-picker": "*",
+				"@clayui/drop-down": "*",
+				"@clayui/empty-state": "*",
+				"@clayui/form": "*",
+				"@clayui/icon": "*",
+				"@clayui/label": "*",
+				"@clayui/layout": "*",
+				"@clayui/link": "*",
+				"@clayui/list": "*",
+				"@clayui/loading-indicator": "*",
+				"@clayui/management-toolbar": "*",
+				"@clayui/modal": "*",
+				"@clayui/multi-select": "*",
+				"@clayui/multi-step-nav": "*",
+				"@clayui/nav": "*",
+				"@clayui/navigation-bar": "*",
+				"@clayui/pagination": "*",
+				"@clayui/pagination-bar": "*",
+				"@clayui/panel": "*",
+				"@clayui/popover": "*",
+				"@clayui/progress-bar": "*",
+				"@clayui/shared": "*",
+				"@clayui/slider": "*",
+				"@clayui/sticker": "*",
+				"@clayui/table": "*",
+				"@clayui/tabs": "*",
+				"@clayui/time-picker": "*",
+				"@clayui/tooltip": "*",
+				"@clayui/upper-toolbar": "*",
+				"clay": "*",
+				"clay-alert": "*",
+				"clay-autocomplete": "*",
+				"clay-badge": "*",
+				"clay-button": "*",
+				"clay-card": "*",
+				"clay-card-grid": "*",
+				"clay-checkbox": "*",
+				"clay-collapse": "*",
+				"clay-component": "*",
+				"clay-data-provider": "*",
+				"clay-dataset-display": "*",
+				"clay-dropdown": "*",
+				"clay-icon": "*",
+				"clay-label": "*",
+				"clay-link": "*",
+				"clay-list": "*",
+				"clay-loading-indicator": "*",
+				"clay-management-toolbar": "*",
+				"clay-modal": "*",
+				"clay-multi-select": "*",
+				"clay-navigation-bar": "*",
+				"clay-pagination": "*",
+				"clay-pagination-bar": "*",
+				"clay-portal": "*",
+				"clay-progress-bar": "*",
+				"clay-radio": "*",
+				"clay-select": "*",
+				"clay-sticker": "*",
+				"clay-table": "*",
+				"clay-tooltip": "*"
+			}
+		}
+	},
+	"rules": [
+		{
+			"description": "Copy static assets",
+			"test": "^assets/.*",
+			"use": [
+				{
+					"loader": "copy-loader"
+				}
+			]
+		},
+		{
```

Comment:
Please have a look at this [relevant comment](https://github.com/liferay/liferay-frontend-projects/issues/543#issuecomment-844984684) when reviewing this.